### ISSUE TITLE
Doesn't compile with latest Debian version

### ISSUE THREAD

**Issue by wizzardx** [Vladar4/nimgame2#3]
Required SDL2 TTF version for this lib is 2.0.14.

This is also where a required symbol for demo1 was added: TTF_GetFontKerningSizeGlyphs:

https://www.libsdl.org/projects/SDL_ttf/

However, the latest Debian version ships with SDL2 TTF lib version 2.0.12.

This makes it a lot harder to play with demo1:

<pre>
$ ./demo1 

could not import: TTF_GetFontKerningSizeGlyphs
</pre>

Also, the SDL2 ttf page says this for linux distros:

"Please contact your distribution maintainer for updates."

Could you perhaps please also include in your docs, a basic howto on how to compile nimgame2 (eg on misc linux distros) against a new enough version of SDL2?

**Comment by Vladar4** [comment]
It looks like v2.0.14 is present in Debian Testing (Stretch) https://packages.debian.org/stretch/libsdl2-ttf-dev
You will need the latest sdl2 v2.0.5 too. See the readme file for https://github.com/Vladar4/sdl2_nim

**Comment by wizzardx** [comment]
Thanks, that worked for me! I can play more with this now.

For other people's reference: First I made sure that the debian stable versions were up to date:

<pre>
$ sudo apt-get install libsdl2-image-dev libsdl2-gfx-dev libsdl2-mixer-dev libsdl2-ttf-dev
</pre>

After then, when there were problems running demo1 like this:

<pre>
could not import: TTF_GetFontKerningSizeGlyphs
</pre>

I added this line to my /etc/apt/sources.list:

<pre>
http://httpredir.debian.org/debian/ testing main contrib non-free
</pre>

Then ran "apt-get update" and "apt-get install libsdl2-ttf-dev"

After then, commented that line out in my sources.list so that I don't make my Debian Stable system more unstable later on.
